### PR TITLE
Add additional thought sharing

### PR DESCRIPTION
**Pull request by gwensmu**
* Participants may share adding
  a patternt to a thought, or reshaping
  a thought
* Labels now show up in the feed to indicate
  how a thought was changed
* Add fixtures and specs to test these changes
* Update Engine version to pull in related changes
* Same changes as https://github.com/cbitstech/think_feel_do_so/commit/041e85fe8c439d111b2ef3c598dd892550814280

[#90925872]

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "rails", "= 4.1.9"
-gem "sprockets-rails", "= 2.1.3"
+gem "sprockets-rails", "= 2.2.2"
```
Comment (ericcf) [inline Gemfile]: @gwensmu why did you update gems as part of this commit?

**Comment by gwensmu** [comment]
@ericcf had to update the social networking gem in another commit to update the styles on moodtech, when i bundled against SN gem, got conflicts. have been using sprockets-rails 2.2.2 because that is what telehealth uses.

**Comment by gwensmu** [comment]
@ericcf sprockets-rails 2.1.3 was causing a problem with some of the font stuff in the telehealth themes when i imported them into moodtech so i bumped everything up to 2.2.2. see pivotal 90940580

**Comment by ericcf** [comment]
Ok, looks good! :panda_face:

**Pull request merged**